Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,144 @@
 ﻿using System;
-
+using System.Text.RegularExpressions;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    class Equation{
+        private string operand1;
+        private string expectedOperand1;
+        private string operand2;
+        private string expectedOperand2;
+        private string operation;
+        private string result;
+        private string expectedResult;
+        private void setOperand1(string value){

```

Comment:
Public string Operand1 {get; private set;} is more clear,easy and readable way of writing setters and getters